test/batch-transaction.ts

```diff
@@ -485,4 +491,147 @@ describe('BatchTransaction', () => {
       assert.strictEqual(identifier.timestamp, TIMESTAMP);
     });
   });
+
+  describe('observability traces', () => {
+    const traceExporter = new InMemorySpanExporter();
+    const sampler = new AlwaysOnSampler();
+
+    const provider = new NodeTracerProvider({
+      sampler: sampler,
+      exporter: traceExporter,
+    });
+    provider.addSpanProcessor(new SimpleSpanProcessor(traceExporter));
+
+    afterEach(() => {
+      traceExporter.reset();
+    });
+
+    const REQUEST = sandbox.stub();
+    const SESSION = {
+      parent: DATABASE,
+      formattedName_: 'abcdef',
+      request: REQUEST,
+    };
+    const ID = '0xdeadbeef';
+    const TIMESTAMP = {seconds: 0, nanos: 0};
+
+    const PARTITIONS = [{partitionToken: 'a'}, {partitionToken: 'b'}];
+    const RESPONSE = {partitions: PARTITIONS};
+
+    beforeEach(() => {
+      batchTransaction.session = SESSION as {} as Session;
+      batchTransaction.id = ID;
+      batchTransaction.observabilityOptions = {tracerProvider: provider};
+
+      REQUEST.callsFake((_, callback) => callback(null, RESPONSE));
+    });
+
+    const GAX_OPTS = {};
+
+    const fakeDirectedReadOptionsForRequest = {
```

Comment:
This is optional param , so better to remove it to keep the test short. 
Same goes for dataBoostEnabled

Reply:
Done, thanks!